extensions/README.md

```diff
@@ -9,6 +9,7 @@
 * [KHR_materials_pbrSpecularGlossiness](2.0/Khronos/KHR_materials_pbrSpecularGlossiness/README.md)
 * [KHR_materials_unlit](2.0/Khronos/KHR_materials_unlit/README.md)
 * [KHR_texture_transform](2.0/Khronos/KHR_texture_transform/README.md)
+* [KHR_XMP](2.0/Khronos/KHR_XMP/README.md)
```

Comment:
Depending on when this pull request is merged, we might want to move this to the **In progress Khronos and multi-vendor extensions** section below - that way the PR can be merged before Khronos ratification is finished.